hip_data_tools/aws/athena.py

```diff
@@ -212,6 +216,24 @@ def get_table_ddl(self, table):
             ddl = ddl + "\n"
         return ddl
 
+    def get_table_data_location(self, table):
+        """
+        Retrieves the table data location s3 using glue meta store
+        Args:
+            table (str): name of the table
+        Returns: a tuple of s3 bucket and key
+        """
+        table = self._get_glue_table_metadata(table)
+        location = table['Table']['StorageDescriptor']['Location']
+        bucket = location.split("/")[2]
+        key = "/".join(location.split("/")[3:])
+        return (bucket, key)
+
+    def _get_glue_table_metadata(self, table):
```

Comment:
there's a fair bit of logic going on here. Should this be a private method? You might wan to refactor

Reply:
changed to a single object creation and method call.